src/core_plugins/kibana/public/home/components/add_data.js

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  KuiCardGroup,
+  KuiCard,
+  KuiCardDescription,
+  KuiCardDescriptionTitle,
+  KuiCardDescriptionText,
+  KuiCardFooter,
+} from 'ui_framework/components';
+
+import {
+  EuiButton,
+  EuiLink,
+  EuiPanel,
+  EuiTitle,
+  EuiSpacer,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+} from '@elastic/eui';
+
+export function AddData({ addBasePath, isCloudEnabled }) {
+
+  const renderCards = () => {
+    const cardStyle = {
+      width: '250px',
+      'minWidth': '200px',
+      'border': 'none'
+    };
+
+    let apmCard;
+    if (!isCloudEnabled) {
+      apmCard = (
```

Comment:
I know we're only refactored this code to be here, but I just realized that APM is hard-coded here, even though it's not a part of OSS Kibana, wasn't there a pluggable/registry-oriented way we can accomplish this?

Reply:
There is no plug-able way to add cards to the home page.

Reply:
Doesn't that mean that APM always shows up even if we don't have x-pack installed?

Reply:
Yep.  APM has an open source side, and you can view the data collected using OSS Kibana dashboards.  The home page tutorial walks you through that, even though there is no APM app inside OSS Kibana (only the x-pack version).  @nreese is working on a PR that will link to either the OSS dashboard, or the APM app, from the tutorial depending on whether x-pack is installed or not (https://github.com/elastic/kibana/pull/16818)